### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by dmytroosypiuk**
- [DEMO LINK](https://mtctxd.github.io/react_person/)

### CODE REVIEW

**Review by krasnobokydima — CHANGES_REQUESTED** [review]
Щось забув додати)

**Comment by mtctxd** [comment]
Поняв......

**Review by krasnobokydima — APPROVED** [review]

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]

File: src/components/Person/Person.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  let parthnerPhrase = `wife's`;
+  let marriedTag = (
+    <p className="Person__partner">I am not married</p>
+  );
```
Comment (Anna-Tkachenko) [inline src/components/Person/Person.jsx:8]: It's better to store only string value, not all tag

File: src/components/Person/Person.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+  if (person.sex === 'f') {
+    parthnerPhrase = `husband's`;
+  }
+
+  if (person.isMarried) {
+    marriedTag = (
+      <p className="Person__partner">
+        {`My ${parthnerPhrase} name is ${person.partnerName}`}
+      </p>
```
Comment (Anna-Tkachenko) [inline src/components/Person/Person.jsx:18]: Same here, don't store tag, just needed string value

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]

**Review by Anna-Tkachenko — APPROVED** [review]

**Pull request closed without merging**